Stack/Opc.Ua.Core/Security/Certificates/SecurityConfiguration.cs

```diff
@@ -59,9 +59,40 @@ public void Validate()
             {
                 throw ServiceResultException.Create(StatusCodes.BadConfigurationError, "ApplicationCertificate must be specified.");
             }
+            // ensure mandatory stores are valid
```

Comment:
Shouldn't these mandatory and later inter-dependent pair relations be formalized somewhere and made more visible, probably even in the ApplicationConfiguration.xsd ?
Just for the sake of "minimizing" confusion during configuration errors ...